src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,109 @@
-export const Pagination = () => {};
+import React from 'react';
+
+import { getNumbers } from '../../utils';
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
```

Comment:
Consider handling the case where 'perPage' is zero to prevent division by zero errors.